packages/fxa-settings/src/pages/ResetPasswordRedesign/ResetPassword/en.ftl

```diff
@@ -1,12 +1,6 @@
 ## ResetPassword start page
 
-# Strings within the <span> elements appear as a subheading.
-# If more appropriate in a locale, the string within the <span>, "to continue to account settings" can stand alone as "Continue to account settings"
-password-reset-heading-w-default-service = Password reset <span>to continue to account settings</span>
-# Strings within the <span> elements appear as a subheading.
-# If more appropriate in a locale, the string within the <span>, "to continue to { $serviceName }" can stand alone as "Continue to { $serviceName }"
-# { $serviceName } represents a product name (e.g., Mozilla VPN) that will be passed in as a variable
-password-reset-heading-w-custom-service = Password reset <span>to continue to { $serviceName }</span>
```

Comment:
Interesting that we're removing this, given the conversation about users sorta being confused by the branding signing up or signing in to an RP. I would expect this to make it worse for password reset if we're going to redirect them at the end of the flow, but maybe the end state is clear enough.

Reply:
Yeah, I verified that this change was intended and Vesta was in favour of simplifying the heading. Perhaps in preparation of the designs that have the relying party logo shown above the card instead of having "continue to ..." text? Also, we still have (for now) have the relying party mentioned on the index page so maybe not necessary to repeat here as well.